docs/modules/ROOT/pages/cicd_concept.adoc

```diff
@@ -0,0 +1,28 @@
+= Continuous Delivery
+
+In order to ensure that code released to production is of the highest possible quality, we recommend
+setting up three separate deployment environments to play different roles in the Continuous Integration 
+/ Continuous Delivery process.
+
+== Target Enviroments
+
+A typical CI/CD framework would deploy images to the following three environments:
+
+=== Development
+
+The development environment is used for internal testing of infrastructure, such as endpoints,
+pipelines, back-end authentication, load tests, and database configuration.
```

Comment:
Can we add a sentence that forbids any external parties relying on DEV? I.e. at BAR we now have the problem that external parties use DEV and we can't properly use it anymore to test changes because they might break stuff...

Reply:
Good point, I have just added a sentence about it.